### PR TITLE
IRC bot more verbose .version response

### PR DESCRIPTION
**Pull request by deutor**
Version response will now include short git revision and DB schema # 
Issue #4354 
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: LibreNMS/IRCBot.php [hunk 1]
```diff
@@ -569,7 +569,13 @@ private function _help($params)
 
     private function _version($params)
     {
-        return $this->respond($this->config['project_name_version'].', PHP: '.PHP_VERSION);
+        $versions       = version_info(false);
+        $schema_version = $versions['db_schema'];
+        $version        = `git rev-parse --short HEAD`;
```
Comment (laf) [inline LibreNMS/IRCBot.php]: We already have: local_sha

You'll just need to cut it down to a shorter version. Better than doing another exec call.

File: LibreNMS/IRCBot.php [hunk 2]
```diff
@@ -571,8 +571,7 @@ private function _version($params)
     {
         $versions       = version_info(false);
         $schema_version = $versions['db_schema'];
-        $version        = `git rev-parse --short HEAD`;
-        $version        = preg_replace('/\s+/', ' ', $version);
+        $version        = substr($versions['local_sha'], 1, 7);
```
Comment (laf) [inline LibreNMS/IRCBot.php]: Shouldn't this be 0, 6 as you want to start at the first char?

**Comment by deutor** [comment]
You're right - substr should start at 0 (I usually work with language where substrings are 1 based). But 0, 7 - as I want first 7 chars just like git rev-parse --short does.

**Comment by laf** [comment]
Great work :)

image

**Pull request merged**